.dockerignore

```diff
@@ -1 +1,46 @@
-.gitignore
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
```

Comment:
Just curious is it possible to use a soft link to `.giignore` file

Reply:
good idea